tests/commit.bats

```diff
@@ -96,3 +96,26 @@ load helpers
   cid=$(buildah from --pull --signature-policy ${TESTSDIR}/policy.json alpine)
   run_buildah commit --signature-policy ${TESTSDIR}/policy.json $cid
 }
+
+@test "commit with name" {
+  buildah from --name nginxc nginx
+  run_buildah --log-level=error commit --signature-policy ${TESTSDIR}/policy.json nginxc commitbyname/nginxbyname
+  cid=$(buildah from localhost/commitbyname/nginxbyname)
+  run_buildah inspect $cid
+  buildah rm nginxc $cid
+  buildah rmi commitbyname/nginxbyname nginx
+}
+
+@test "commit to docker-distribution" {
+  run which podman
+  if [ $status -ne 0 ]; then
+    skip "Can not find podman command skip the test"
+  fi
+  podman run -d -p 5001:5000 --name registry registry:2
```

Comment:
WHOA! You cannot do this: there is already a registry running, and with this name too. It is an implied requirement for the buildah bats tests.